doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+List of compute nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_infos | List | Yes | a list of cluster information|
+<p align="center"><b>Table 5-3-8-1:</b> Storage information.</p>
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster_name | String | Yes | indicate the cluster name|
+| ceph_iscsi_gateway | String | Yes | splitted by semicolon if more than one. e.g 2409:8086:8412:10a::1;2409:8086:8412:10a::2|
+| ceph_manage_plane_address_segment | String | Yes | The address segment of ceph management plane. e.g 2409:8086:8412:100::/64|
+| ceph_storage_plane_address_segment | String | Yes | The address segment of ceph storage plane. e.g 2409:8086:8412:10a::/64|
+| ceph_os_user | String | Yes | the user of ceph OS. |
+| ceph_os_password | String | Yes | the user's password of ceph OS. |
+| ceph_manage_plane_float_ip | String | Yes |  |
+| ceph_network_manage_plane_float_ip | String | Yes |  |
+| zms_network_manage_admin_password | String | Yes |  |
+| ceph_device_name | String | Yes |  |
+| ceph_iscsi_device_name | String | Yes |  |
+| glance_pool_name | String | Yes |  |
+| nova_pool_name | String | Yes |  |
+| belong_to_az_one | String | Yes |  |
+| cinder_infos | List | Yes |a list of cinder infos which are cinder pool name and backend info |
+
+<p align="center"><b>Table 5-3-8-2:</b> Cluster information.</p>
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cinder_pool_name | String | Yes | cinder pool name|
+| cinder_backend_name | String | Yes | backend name cinder connected|
+<p align="center"><b>Table 5-3-8-3:</b> Cinder information.</p>
+
+### 5.3.9 NTP server informations
+primary and backup NTP server information.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| master_server_ip | String | Yes | |
+| master_server_timezone | String | Yes | |
+| backup_server_ip | String | No | |
+| backup_server_timezone | String | No | |
+
+<p align="center"><b>Table 5-3-9:</b> NTP server information.</p>
+
+### 5.3.10 DNS server informations
```

Comment:
There should be a section to capture proxy IP.

Reply:
sure,I will add at last, could you give other parameters besides, ip, user,password?

Reply:
We should be good with IP/port, user and password.